pro/src/components/OnboardingOffersChoice/OnboardingOffersChoice.tsx

```diff
@@ -0,0 +1,56 @@
+import { Button } from 'ui-kit/Button/Button'
+
+import './OnboardingOffersChoice.scss'
```

Comment:
On a plutôt l'habitude de faire l'import comme ça, et dans l'utiliser avec un `className={styles['card']}`
```suggestion
import stryles from './OnboardingOffersChoice.scss'
```